generate-changelog/action.yaml

```diff
@@ -33,9 +36,13 @@ runs:
         git cliff --output CHANGELOG.md --config ${{ inputs.git-cliff-config }} \
           -vv --strip header ${{ inputs.git-cliff-args }}
 
-        r=$(cat CHANGELOG.md)
+        # replace PR number to URL. ex) "#400" -> "http://github.com/test/pull/400"
+        ${GITHUB_ACTION_PATH}/replace_pr_number.sh CHANGELOG.md ${{ inputs.target-repository }} > CHANGELOG_replaced.md
```

Comment:
I feel just using `sed` is enough for this use case.
Do you need to use `awk` and a script file?

Reply:
I have no idea against using `sed`. Using `awk` and script file is the limit of my ability.
Do you have any idea?

Reply:
How about this?

```sh-session
$ changelog=$(git-cliff --current)
$ NAME=https://github.com/autowarefoundation/autoware-github-actions
$ echo "$changelog" | sed -r "s|\(#([0-9]+)\)|($NAME/pull/\1)|g"
# Changelog

All notable changes to this project will be documented in this file.

## [1.1.0] - 2022-03-04

### <!-- 00 -->Features

- [deploy-docs] Add an option to create the 'latest' alias (https://github.com/autowarefoundation/autoware-github-actions/pull/93)

### <!-- 06 -->Tests

- Fix URL of .clang-tidy (https://github.com/autowarefoundation/autoware-github-actions/pull/91)

### <!-- 09 -->Chores

- Sync sync-files.yaml (https://github.com/autowarefoundation/autoware-github-actions/pull/92)

<!-- generated by git-cliff -->
```

Reply:
Thank you for your solution. That’s new for me. I'll fix

Reply:
Please replace the name of the variable `NAME` with an appropriate one. :+1: